### ISSUE TITLE
I need whoever is in charge of writing the function for splitting train and test set in order to fully modularize my function

### ISSUE THREAD

**Issue by alexzhang0825** [UBC-MDS/Red-Wine-Quality-Prediction#33]
The model selection and hyperparameter section tuning depend on the result from the train test split, which I believe at this point should be saved as a standalone .csv file, but I am not in charge of that section so I need you guys to complete that part so that I can fully modularize my function.

**Comment by alexzhang0825** [comment]
I've managed to do it using manually exported train test split datas
